var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -50,6 +52,7 @@ class Scotch(CMakePackage, MakefilePackage):
         when="@7.0.1",
         description="Link error handling library to libscotch/libptscotch",
     )
+    variant("fismahigh", default=False, description="Apply patching for FISMA-high compliance")
```

Comment:
Do we want to add a patch to comply to FISMA? I mean, potentially every package could have that...

Reply:
I found this https://github.com/spack/spack/pull/36470#issuecomment-1485762539 and kind of agree with @skosukhin that this should be the concern of a custom repo...